src/CallbackResolver.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace DI\Bridge\Silex;
+
+use Invoker\CallableResolver;
+use Invoker\Exception\NotCallableException;
+
+/**
+ * Class CallbackResolver
+ * @package DI\Bridge\Silex
```

Comment:
No need for the `@package` annotation which is covered by the PHP namespace. Also could you update the description to explain why this class exists? e.g. `This alternative resolver uses the generic Invoker to support PHP-DI's extended callable syntax`